src/create-app.ts

```diff
@@ -34,10 +35,20 @@ export async function createApp(starter: Starter, projectName: string, autoRun:
   loading.stop(true);
 
   const time = printDuration(Date.now() - startT);
-  console.log(`${green('✔')} ${bold('All setup')} ${onlyUnix('🎉')} ${dim(time)}
+  let didGitSucceed = initGitForStarter(projectName);
+
+  if (didGitSucceed) {
+    console.log(`${green('✔')} ${bold('All setup')} ${onlyUnix('🎉')} ${dim(time)}`);
```

Comment:
This little `onlyUnix` is interesting...it's implementation is the exact opposite of what I'd expect - https://github.com/ionic-team/create-stencil/blob/690a33025bb3cf4384d922df502265e90b31f144/src/utils.ts#L64-L66

Punting on fixing that/messing with different OS terminals for another day...I followed the pre-existing patterns of using it for now